packages/aws-rfdk/lib/lambdas/nodejs/pad-efs-storage/handlers.ts

```diff
@@ -0,0 +1,192 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* eslint-disable no-console */
+
+import {
+  promises as fsp,
+} from 'fs';
+import {
+  join,
+} from 'path';
+import {
+  LambdaContext,
+} from '../lib/aws-lambda';
+import {
+  determineNextSequentialFilename,
+  diskUsage,
+  ensureIsDirectory,
+  listNumberedFiles,
+  nextSequentialFile,
+  writePaddingFile,
+} from './filesystem-ops';
+
+/**
+ * Default filesize for lambda operation, in MB.
+ * External code that calls this assumes that this is exactly 1GB=1000MB
+ */
+var FILESIZE: number = 1000;
+
+/**
+ * Provided solely for the purpose of testing to shrink the default file size from 1GB
+ * to something smaller.
+ * @param filesize Desired filesize in MB. Must be a multiple of 10
+ */
+export function setDefaultFilesize(filesize: number) {
+  FILESIZE = filesize;
+}
+
+/**
+ * Local helper to extract the desiredPadding field from the input event, and validate the input.
+ * @param event Event object passed to the lambda handler.
+ * @returns Value of desiredPadding or throws if invalid.
+ */
+function getDesiredPadding(event: { [key: string]: string }): number {
+  const desiredPadding = parseInt(event.desiredPadding);
+  if (isNaN(desiredPadding)) {
+    throw new Error(`Could not parse 'desiredPadding' field of the given event: ${event.desiredPadding}`);
+  }
+  return desiredPadding;
+}
+
+/**
+ * Local helper to extract the mountPoint field from the input event, and validate the input.
+ * @param event Event object passed to the lambda handler.
+ * @returns Value of mountPoint or throws if invalid.
+ */
+function getMountPoint(event: { [key: string]: string }): string {
+  const mountPoint = event.mountPoint;
+  if (!mountPoint) {
+    throw new Error(`Invalid mount point given in event: ${mountPoint}`);
+  }
+  return mountPoint;
+}
+
+/**
+ * Add numbered files (e.g. 00000, 00001) of a given size to a filesystem.
+ * Note: exported so that we can test it.
+ * @param numFilesToAdd How many numbered files to add.
+ * @param filesize Size, in MB, of the files to create.
+ * @param mountPoint Directory in which to create the directory.
+ */
+export async function growFilesystem(numFilesToAdd: number, filesize: number, mountPoint: string): Promise<void> {
+  // We find the highest numbered file created thus far, and start adding numbered files after it.
+  var filename: string = await determineNextSequentialFilename(mountPoint);
```

Comment:
```suggestion
  let filename: string = await determineNextSequentialFilename(mountPoint);
```

Same goes for other uses of `var` that can be block-scoped

Reply:
Good point!